proto/internalapi/central/compliance_operator.proto

```diff
@@ -125,28 +137,52 @@ message ComplianceOperatorCheckResultV2 {
         INCONSISTENT   = 7;
     }
 
-    enum RuleSeverity {
-        UNSET_RULE_SEVERITY = 0;
-        UNKNOWN_RULE_SEVERITY = 1;
-        INFO_RULE_SEVERITY = 2;
-        LOW_RULE_SEVERITY = 3;
-        MEDIUM_RULE_SEVERITY = 4;
-        HIGH_RULE_SEVERITY = 5;
+    string                         id           = 1;
+    string                         check_id     = 2;
+    string                         check_name   = 3;
+    string                         cluster_id   = 4;
+    CheckStatus                    status       = 5;
+    ComplianceOperatorRuleSeverity severity     = 6;
+    string                         description  = 7;
+    string                         instructions = 8;
+    map<string, string>            labels       = 9;
+    map<string, string>            annotations  = 10;
+    google.protobuf.Timestamp      created_time = 11;
+    string                         standard     = 12;
+    string                         control      = 13;
+    string                         scan_name    = 14;
+    string                         suite_name   = 15;
+}
+
+// ComplianceOperatorProfileV2 is a message from Sensor (to Central) representing a compliance check profile.
+// Next tag: 10.
+message ComplianceOperatorProfileV2 {
+    string              id              = 1;
+    string              profile_id      = 2;
+    string              name            = 3;
+    string              version         = 4;
+    string              profile_version = 5;
+    map<string, string> labels          = 6;
+    map<string, string> annotations     = 7;
+    string              description     = 8;
+    message Rule {
+        string rule_name       = 1;
     }
+    repeated Rule       rules       = 9;
+}
 
```

Comment:
CO Profile CRD also contains list of Values, which is all variable that present in that profile, wondering if we need those?

Reply:
@Vincent056 Is this only applicable for tailored profiles?

Reply:
Variables are being used to evaluate the rules dynamically based on the values of them. Variables comes with a default values, and each profile might have a different values for a variable. Tailored profile can be used to set values for those variables.

Reply:
Sounds good.  I think we will wait for doing tailored profiles sometime post MVP.